programs/uxd/src/events.rs

```diff
@@ -96,6 +96,19 @@ pub struct SetDepositoryMintingDisabledEvent {
     pub minting_disabled: bool,
 }
 
+/// Event called in [instructions::edit_*_depository::handler].
```

Comment:
nit: this diff should disappear when rebasing with master

Reply:
yes rebasing